src/python/bot/tasks/setup.py

```diff
@@ -63,33 +63,35 @@ def _copy_testcase_to_device_and_setup_environment(testcase,
 
   # The following steps need privileged job access.
   job_type_has_privileged_access = environment.get_value('PRIVILEGED_ACCESS')
-  if job_type_has_privileged_access:
```

Comment:
There should be no functional changes here, just noticed that it was formatted in a way that would have been better served by an early bailout.